nixos/modules/profiles/base.nix

```diff
@@ -7,7 +7,6 @@
   # Include some utilities that are useful for installing or repairing
   # the system.
   environment.systemPackages = [
-    pkgs.w3m-nographics # needed for the manual anyway
```

Comment:
I can't see any motivation to remove w3m from $PATH.

Reply:
Dropped.